### DISCUSSION TITLE
What is the correct action for project-commit create?

### DISCUSSION THREAD

**Discussion by JohannesM** [python-gitlab/python-gitlab#2145] [Q&A] [answered]
```bash
	gitlab project-commit create \
		--project-id 123 \
		--branch 'main' \
		--commit-message '[skip ci] add README.rst' \
		--actions '{"action": "create","file_path": "README.rst","content": "test"}'
```

Error: Impossible to create object (400: actions is invalid)

What is the correct value for actions here?

**Comment by nejch** [comment, marked as answer]
@JohannesM sorry for the delay. I think our CLI currently does not yet handle this and sends it as a string rather than proper JSON. Feel free to create an issue from this discussion and we can track it.

**Comment by spaenigs** [comment]
What is the current state? Does the CLI handle this now?
